stistools/__init__.py

```diff
@@ -16,9 +16,10 @@
 from . import tastis
 from . import ctestis
 from . import defringe
+from . import poisson_err
 
 # These lines allow TEAL to print out the names of TEAL-enabled tasks
 # upon importing this package.
 import os
 from stsci.tools import teal
-teal.print_tasknames(__name__, os.path.dirname(__file__))
\ No newline at end of file
+teal.print_tasknames(__name__, os.path.dirname(__file__))
```

Comment:
Was this breaking the install?  I recall @stscirij talking about removing `teal` code...

Reply:
No, I think install was fine. I just re-installed the latest version of stenv and everything went smoothly. When importing stistools, I get the usual NMPFIT and GFIT deprectation warnings and then the usual TEAL package note ("The following tasks...").